gammapy/spectrum/tests/test_spectrum_fit.py

```diff
@@ -0,0 +1,57 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.tests.helper import pytest
+from numpy.testing import assert_allclose
+
+from gammapy.spectrum import run_spectrum_extraction_using_config
```

Comment:
Use relative imports within Gammapy ... this needs to be changed in several files in this PR.